doc/1.4/language.md

```diff
@@ -3300,29 +3355,156 @@ after h -> (x, y): my_callback(i, y + 1.5, false)
 ```
 as the message component parameter `y` is used, but not as a direct argument.
 
-For both forms of `after`, to allow the suspended method call to be represented
-in a checkpoint, `after` statements may only be performed with methods that have
-no return values, and where each input parameter of the method must be of
-[*serializable type*](#serializable-types), unless that input parameter receives
-a message component.
+#### Immediate After Statements
+<pre>
+after: <em>method</em>(<em>e1</em>, ... <em>eN</em>);
+</pre>
 
-This means that `after` statements cannot be used with methods that e.g. have
-pointer input parameters, unless the arguments for those input parameters are
-message component parameters of the `after`.
+In this form, the specified point in the future when control is given back to
```

Comment:
Grammatical artifact from too many revisions, should be "the specified point in the future is when"